java/HealthVault Mobile/HVSample/src/main/res/layout/current_record_item.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There is no need for a complex layout time like relative layout if we only have a single control inside. FrameLayout might be more suitable or even use of the "<merge>" tag